GLMakie/src/glwindow.jl

```diff
@@ -107,6 +107,11 @@ function GLFramebuffer(fb_size::NTuple{2, Int})
         internalformat = GL_DEPTH24_STENCIL8,
         format = GL_DEPTH_STENCIL
     )
+
+    # Holds the (temporary) render of a scene
+    color_buffer = Texture(
+        RGBA{N0f8}, fb_size, minfilter = :nearest, x_repeat = :clamp_to_edge
+    )
     # Order Independent Transparency
     HDR_color_buffer = Texture(
         RGBA{Float16}, fb_size, minfilter = :linear, x_repeat = :clamp_to_edge
```

Comment:
Not right now. I'm also not sure what you mean with precise renders... 
As it is now, every scene effectively clears a section of this buffer, then does all the rendering on it and runs all the post processors with it. So it's the same as `colorbuffer(detached_sub_scene)` after that scene is done rendering, except with a bunch of old junk around it. At the end of a frame it maybe overwritten by another scene.
I could add a `colorbuffer(screen, scene)` that only renders a selected scene if that's what you want.

Reply:
The `colorbuffer(screen, scene)` thing is exactly what I need for CairoMakie :D since cropping the scene by its bbox also includes axis artifacts, etc.